dask_cuda/cli/dask_cuda_worker.py

```diff
@@ -277,6 +277,13 @@
     help="""Use a different class than Distributed's default (``distributed.Worker``)
     to spawn ``distributed.Nanny``.""",
 )
+@click.option(
+    "--pre-import",
+    default=None,
```

Comment:
Should we not include `cudf` and `rmm` by default?

If not, because we don't want to assume all dask-cuda users are cudf users, this would be a point in favor of thinking more about the concept of dask-cuda "profiles" which include the defaults most users will want without knowing they want them.

Reply:
Building a [RAPIDS worker plugin](http://distributed.dask.org/en/stable/plugins.html#worker-plugins) would make sense.  But yes, for now dask-cuda is still general purpose and we shouldn't include it by default.

Reply:
Yeah we do have people come by periodically who are simply using Dask-CUDA alone. They might have issues if we added other hard requirements like these

That said, maybe we could include an example in the docs?

Reply:
This is the example that is in the argument description itself, see https://github.com/rapidsai/dask-cuda/pull/854/files#diff-b3d47096fc004d0fe86d1630b46845c70e79a9914b065ba2b52c4fc073dd1621R283-R285 and https://github.com/rapidsai/dask-cuda/pull/854/files#diff-245fed94d1f88f78dbd6898226b22c3cb2084eb9e8e7c8994beee67c44461f6aR169-R171 . I'm not sure I see what more is being suggested we do here.

Reply:
Yeah agree docs are sufficient already then

Maybe @randerzander could share a bit more of what he is looking for with Dask-CUDA profiles in a new issue and we could discuss that? 😉